modules/test/playwright/tests/workspaces/liferay-customer-workspace/tests/general/unit/projectUserPermission.spec.ts

```diff
@@ -0,0 +1,190 @@
+	await mockProvisioningApiAssignUser(page);
+
+	const account =
+		await apiHelpers.headlessAdminUser.getAccountByExternalReferenceCode(
+			accountExternalReferenceCode
+		);
+
+	await apiHelpers.headlessAdminUser.assignUserToAccountByEmailAddress(
+		account.id,
+		['test@liferay.com']
+	);
+
+	const rolesResponse = await apiHelpers.headlessAdminUser.getAccountRoles(
+		account.id
+	);
+
+	const accountAdministratorRole = rolesResponse?.items?.filter((role) => {
+		return role.name === 'Account Administrator';
+	});
+
+	await apiHelpers.headlessAdminUser.assignAccountRoles(
+		accountExternalReferenceCode,
+		accountAdministratorRole[0].id,
+		'test@liferay.com'
+	);
+});
+
+test.describe('Project User Permission', () => {
+	test('User can only see things related to User role', async ({
+		apiHelpers,
+		customerApiHelpers,
+		homePage,
+		page,
+		projectAttachmentsPage,
+		projectOverviewPage,
+		projectPaaSPage,
+		projectTeamMembersPage,
+	}) => {
+		await projectOverviewPage.goto(accountExternalReferenceCode);
+
+		const accountFlag = await customerApiHelpers.getAccountFlag(
+			accountExternalReferenceCode
+		);
+
+		if (accountFlag === undefined) {
+			await expect(
+				page.getByRole('button', {name: 'Start Project Setup'})
+			).toBeVisible();
+		}
+
+		await projectTeamMembersPage.goto(accountExternalReferenceCode);
+
+		await projectTeamMembersPage.inviteButton.click();
+
+		await projectTeamMembersPage.firstNameField.fill('testfirst');
+
+		await projectTeamMembersPage.lastNameField.fill('testlast');
+
+		userEmailAddress = getRandomString() + '@liferay.com';
+
+		await projectTeamMembersPage.emailField.fill(userEmailAddress);
+
+		await projectTeamMembersPage.roleSelect.click({force: true});
+
+		await projectTeamMembersPage.userRoleOption.click();
+
+		await projectTeamMembersPage.applyButton.click();
+
+		await projectTeamMembersPage.sendInvitationsButton.click();
+
+		await projectTeamMembersPage.goto(accountExternalReferenceCode);
+
+		await expect(page.getByText(userEmailAddress)).toBeVisible();
+
+		const userAccount =
+			await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+				userEmailAddress
+			);
+
+		await apiHelpers.headlessAdminUser.patchUserAccount(userAccount, {
+			currentPassword: 'test',
+			password: 'test',
+		});
+
+		await customerPerformUserSwitch(page, userEmailAddress);
+
+		await expect(
+			page.getByRole('heading', {name: 'Test Account Liferay PaaS'})
+		).toBeVisible();
+
+		expect((await homePage.projectCard.all()).length).toBe(1);
+
+		await projectOverviewPage.goto(accountExternalReferenceCode);
+
+		await expect(projectOverviewPage.heading).toBeVisible();
+
+		await expect(projectOverviewPage.paasHeading).toBeVisible();
+
+		await expect(projectOverviewPage.subscriptionCard).toBeVisible();
+
+		await projectPaaSPage.goto(accountExternalReferenceCode);
+
+		await expect(projectPaaSPage.heading).toBeVisible();
+
+		await expect(projectPaaSPage.projectName).toBeVisible();
```

Comment:
we dont need to look for the project name here

Reply:
Done!